client/app/lib/flux/teams/actions.coffee

```diff
@@ -315,21 +316,26 @@ deleteApiToken = (apiTokenId) ->
   reactor.dispatch actions.DELETE_API_TOKEN_SUCCESS, { apiTokenId }
 
 
-addApiToken = (apiToken) ->
+addApiToken = ->
 
   { reactor } = kd.singletons
-  reactor.dispatch actions.ADD_API_TOKEN_SUCCESS, { apiToken }
+  remote.api.JApiToken.create (err, apiToken) ->
+
+    return showError err  if err
```

Comment:
same here, let's remove this from here, and put it into the promise's rejection handler.